src/ProcessedStory.ts

```diff
@@ -128,6 +138,8 @@ const runOptionsSchema = zod.optional(
   }),
 );
 
+const configSchema = zod.object({}).passthrough().optional();
```

Comment:
Decided not to try to validate that the individual properties here are correct (which we do for `runOptions`, but I might change later).

Seems like more to maintain than I really want to, and typescript does a great job already of doing this (for people using typescript).